docker-compose.yml

```diff
@@ -26,3 +26,9 @@ services:
             - ./api/src:/api/src
         ports:
             - 3000:3000
+    he_twitter_miner:
+        build: .\he_twitter_miner
+        volumes:
+        - ${HOST_HE_TWITTER_MINER_SRC_DIR}\he_twitter_miner\src:/usr/src/app
+        - ${HOST_HE_TWITTER_MINER_SRC_DIR}\config:/usr/src/app/config
```

Comment:
Are you sure? I was looking for some kind of bug referencing this fact, and they all seemed old.